RUFAS/graph_generator.py

```diff
@@ -525,10 +528,12 @@ def _log_non_numerical_data(
 
     def _draw_graph(
         self,
-        graph_type: str | list[str],
-        data: Dict[str, List[int | float]],
-        selected_variables: Optional[List[str]] = None,
+        graph_type: str,
+        data: dict[str, list[int | float]],
+        selected_variables: list[str],
+        ax: Axes,
```

Comment:
```suggestion
        ax: Axes | None,
```
Does this need to be None as well?

Reply:
If this is a reference to the docstring param notes, I removed the `default None` from the Axes section. Or were you seeing a warning in VS Code/MyPy somewhere about it? I think because of the way it's set up in the parent function, it wouldn't ever be None.

Reply:
Gotcha, it was a reference to the docstring. I didn't see any warnings about it